functions/Run_TA.m

```diff
@@ -191,7 +197,6 @@
             % saving preprocessed input data as 4D nifti file
             WriteInformation(fid,'Saving preprocessed fMRI 4D input (TC)...');
             save4Dnii(resultsPath,'inputData','TC',TC,param.fHeader.fname,param.mask,param.Dimension);
-            save4Dnii(resultsPath,'inputData','STD_MAP',STD_MAP,param.fHeader.fname,param.mask,param.Dimension);
```

Comment:
If detrending is not done this causes an error

Reply:
This is not code I wrote. This is something for the iCAPs developers to fix in my opinion.

Reply:
It may be if you want we can ommit it and reported to iCAPs but its gonna be quite annoying since it stops the execution of the rest of the script

Reply:
Let's do it on a different PR then.